client/my-sites/upgrades/domain-management/dns/txt-record.jsx

```diff
@@ -55,7 +55,7 @@ const TxtRecord = React.createClass( {
 						name="data"
 						onChange={ this.props.onChange }
 						value={ this.props.fieldValues.data }
-						placeholder={ this.translate( 'e.g. a=b; verification-key=something', { context: 'TXT DNS Record', textOnly: true } ) } />
+						placeholder={ this.translate( 'e.g. %(example)s', { args: { example: 'a=b; verification-key=something' } } ) } />
```

Comment:
Usually you would just paste in a string for example for your SPF record, I think it's ok if the example is in English. It is a technical record and it would be weird to me to see this translated.